src/window.rs

```diff
@@ -1106,6 +1106,25 @@ impl Window {
     self.window.theme()
   }
 
+  /// Sets the theme for this window.
+  ///
+  /// ## Platform-specific
+  ///
```

Comment:
We should mention that on Linux and macOS, this changes the whole application and not just this window